evaluation/metrics/train_file_metrics.py

```diff
@@ -13,12 +14,14 @@
 from eval_config import eval_config
 
 
-s3_client = boto3.client(
-    "s3",
-    aws_access_key_id=eval_config["dataperf_aws_access_key_id"],
-    aws_secret_access_key=eval_config["dataperf_aws_secret_access_key"],
-    region_name=eval_config["dataperf_aws_region"],
-)
+@functools.lru_cache()
```

Comment:
This seems like it will be useful, but I'm a bit afraid to change anything related to dataperf unless you were able to test these changes. I guess it doesn't seem like this will break anything — if you were able to test this change we can definitely leave it, but if not, maybe we could revert it to how it was?

Reply:
yeah it is supposed to be a different aws account I think